Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ApplePayTokenizationViewModel.swift

```diff
@@ -303,55 +345,144 @@ class ApplePayTokenizationViewModel: PaymentMethodTokenizationViewModel {
             }
         }
     }
+
+    func getShippingMethodsInfo() -> ShippingMethodsInfo {
+        guard let options = PrimerAPIConfigurationModule
+            .apiConfiguration?
+            .checkoutModules?
+            .first(where: { $0.type == "SHIPPING"})?
+            .options as? ShippingMethodOptions else {
+            return .init(shippingMethods: nil, selectedShippingMethodOrderItem: nil)
+        }
+
+        var factor: NSDecimalNumber
```

Comment:
[nit]: There are currencies with 0, 2 and 3 minor units. I'd suggest to check if we can create factor like:

factor = pow(10, currency.decimalDigits)

Reply:
Thanks - created a [ticket](https://primerapi.atlassian.net/browse/ACC-4144) for this since it might be a can of worms (we are now matching existing functionality in lineItems)